frontend/taipy-gui/base/src/packaging/taipy-gui-base.d.ts

```diff
@@ -0,0 +1,93 @@
+// Generated by dts-bundle-generator v7.2.0
+
+import { Socket } from 'socket.io-client';
+
+export type ModuleData = Record<string, VarName>;
+export type VarName = Record<string, VarData>;
+export interface VarData {
+    type: string;
+    value: unknown;
+    encoded_name: string;
+}
+declare class DataManager {
+    _data: Record<string, unknown>;
+    _init_data: ModuleData;
+    constructor(variableModuleData: ModuleData);
+    init(variableModuleData: ModuleData): ModuleData;
+    getEncodedName(varName: string, module: string): string | undefined;
+    getName(encodedName: string): [
+        string,
+        string
+    ] | undefined;
+    get(encodedName: string): unknown;
+    getInfo(encodedName: string): VarData | undefined;
+    getDataTree(): ModuleData;
+    getAllData(): Record<string, unknown>;
+    update(encodedName: string, value: unknown): void;
+}
+export type WsMessageType = "A" | "U" | "DU" | "MU" | "RU" | "AL" | "BL" | "NA" | "ID" | "MS" | "DF" | "PR" | "ACK" | "GMC" | "GDT" | "AID" | "GR";
+export interface WsMessage {
+    type: WsMessageType | str;
+    name: string;
+    payload: Record<string, unknown> | unknown;
+    propagate: boolean;
+    client_id: string;
+    module_context: string;
+    ack_id?: string;
+}
+export declare const sendWsMessage: (socket: Socket | undefined, type: WsMessageType | str, name: string, payload: Record<string, unknown> | unknown, id: string, moduleContext?: string, propagate?: boolean, serverAck?: (val: unknown) => void) => string;
+export declare abstract class WsAdapter {
+    abstract supportedMessageTypes: string[];
+    abstract handleWsMessage(message: WsMessage, app: TaipyApp): void;
+}
+export type OnInitHandler = (taipyApp: TaipyApp) => void;
```

Comment:
... with respect to the previous comment, these would therefore return a Boolean value.